examples/combined_ODNP.py

```diff
@@ -309,13 +309,26 @@
             os.remove("temp_ODNP.h5")
             DNP_data.hdf5_write(filename, directory=target_directory)
             final_log.append(
-                "if I got this far, that probably worked -- be sure to move/rename temp_ODNP.h5 to the correct name!!"
+                "if I got this far, that probably worked -- be sure to move/rename temp_ODNP.h5 to the correct name!!")
     logger.info("\n*** FILE SAVED IN TARGET DIRECTORY ***\n")
-    logger.debug(strm("Name of saved data", echo_data.name()))
+    logger.debug(strm("Name of saved data", DNP_data.name()))
     # }}}
     # {{{run IR
+    last_dB_setting = 10
     for j, this_dB in enumerate(T1_powers_dB):
-        p.set_power(this_dB)
+        # {{{ make small steps in power if needed
+        if j == 0:
+            p.set_power(this_dB)
+        else:    
+            if this_dB - last_dB_setting > 3:
+                smallstep_dB = last_dB_setting + 2
+                while smallstep_dB + 2 < this_dB:
+                    p.set_power(smallstep_dB)
+                    smallstep_dB += 2
+                    last_dB_setting = this_dB
+                p.set_power(this_dB)
+        last_dB_setting = this_dB    
```

Comment:
This line should be at the same level as the previous line, and the line two up should be deleted